storybook/stories/components/Tab/Tab.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from  "prop-types";
+import { Tab as MuiTab, Box } from "@mui/material";
+import Tabs from '@mui/material/Tabs';
+import Typography from '@mui/material/Typography';
+
+function TabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+export const Tab = ({ textColor, disabled, ...rest }) => {
+  const [value, setValue] = React.useState(0);
```

Comment:
Instead of React.useState, can we directly import {useState} from "react" and directly use it as `useState(0)` ?
It's considered as good practice mentioned on official react docs.

Reply:
To encourage good practices, this would be great to refactor `React.useState` to simply `useState`

Reply:
Yep, shall we create a new issue ticket for this @senali-d ? for the overall refactor throughout the codebase?